src/submission/models.py

```diff
@@ -1386,6 +1386,7 @@ class FrozenAuthor(models.Model):
 
     institution = models.CharField(max_length=1000)
     department = models.CharField(max_length=300, null=True, blank=True)
+    frozen_biography = models.TextField(null=True, blank=True, verbose_name=_('Biography'))
```

Comment:
@joemull this change should require a migration, you can generate one with `python src/manage.py makemigrations submission`